gamedata.py

```diff
@@ -896,7 +896,7 @@ def description(self) -> str:
         return self._description
 
     def escaped_description(self) -> str:
-        return self.description().replace("\n", "<br/>").replace("'", "\\'")
+        return self.description().replace("\n", "<br>").replace("'", "\\'")
```

Comment:
html strips the / from the <br/>, so it made equating it incorrect. Could do regex or some other way to do a check, but honestly this was easiest and doesn't affect anything.